src/lib/operators/jit_operator/operators/jit_aggregate.cpp

```diff
@@ -219,6 +219,8 @@ void JitAggregate::add_aggregate_column(const std::string& column_name, const Ji
       break;
     case AggregateFunction::CountDistinct:
       Fail("Aggregate function count distinct not supported");
+    case AggregateFunction::StdDevSamp:
+      Fail("Aggregate function standard deviation sample is not supported.");
```

Comment:
Having this here means that the `JitAwareLQPTranslator` needs to be adjusted to not abort if this AggregateFunction is encountered. 

`JitAwareLQPTranslator::_expression_is_jittable()` should use a whitelist of supported AggregateFunctions, instead of a blacklist.

Reply:
https://github.com/hyrise/hyrise/blob/3403d5f29d1e3f9d0670e3adb93c8cc39caaae53/src/lib/logical_query_plan/jit_aware_lqp_translator.cpp#L433-L464